client/src/features/projects/projectMigration.api.ts

```diff
@@ -0,0 +1,21 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/dist/query/react";
+import { Project } from "../projectsV2/api/projectV2.api";
+
+export const projectMigrationApi = createApi({
+  reducerPath: "renku_v1_projects",
+  baseQuery: fetchBaseQuery({ baseUrl: "/ui-server/api" }),
```

Comment:
1. No `/ui-server` prefix
2. Use `/api/data` here.
```suggestion
  baseQuery: fetchBaseQuery({ baseUrl: "/api/data" }),
```

Reply:
sure, solved here [f147103](https://github.com/SwissDataScienceCenter/renku-ui/pull/3527/commits/f1471032d944adf9d773176b2550c2e6f4820dfe)